rb/lib/htmlproofer/enforce_img_https.rb

```diff
@@ -0,0 +1,17 @@
+class EnforceImgHTTPS < ::HTMLProofer::Check
+  def run
+    @html.css('img').each do |node|
+      img = create_element(node)
```

Comment:
If that website was run over an insecure connection (HTTP) and included HTTPS images, it wouldn’t give mixed-content errors.
If that website is run in HTTPS and has hardcoded HTTPS images, no issue either.
So I don’t see the need to support protocol-free URLs here, as it seems the only valid use case (HTTP hotlinks in an HTTPS page) are what we want to guard against.